testing/run-scripts/image_make.sh

```diff
@@ -65,25 +79,26 @@ EXPOSE 9999
 EXPOSE 5900
 EOF
 
-# load-zork.sh needs a copy of Zork in:
-#   /test/src/uproxy-lib/build/dist/samples/zork-{chrome|firefox}app.
-# Unless -p was specified, we need to pull it down.
-if [ "$GIT" = true ]
-then
-  cat <<EOF >> $TMP_DIR/Dockerfile
+  # load-zork.sh needs a copy of Zork in:
```

Comment:
This is actually so it's still part of the else that starts on line 67 - I think it's aligned throughout but let me know if I missed a spot.